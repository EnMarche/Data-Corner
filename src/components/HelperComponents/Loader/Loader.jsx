import React from 'react';

const Loader = () => (
    <span className="spin-loader">
        <i className="fas fa-spinner" />
    </span>
);

export default Loader;
