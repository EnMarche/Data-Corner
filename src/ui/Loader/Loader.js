import { CircularProgress } from '@mui/material'

const Loader = () => <CircularProgress size={16} />

export default Loader
