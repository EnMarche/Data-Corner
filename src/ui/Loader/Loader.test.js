import { render } from '@testing-library/react'
import Loader from 'ui/Loader/Loader'

describe('Loader', () => {
  it('displays Loader', () => {
    const { container } = render(<Loader />)

    expect(container).toMatchSnapshot()
  })
})
