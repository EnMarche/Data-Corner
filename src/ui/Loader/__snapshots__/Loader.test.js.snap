// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader displays Loader 1`] = `
<div>
  <span
    class="spin-loader"
  >
    <i
      class="fas fa-spinner"
    />
  </span>
</div>
`;
